.prettierignore

```diff
@@ -0,0 +1,4 @@
+.cache
```

Comment:
These prettier files are related to a program that ensures certain style conventions.